### ISSUE TITLE
ValueError: invalid literal for int() with base 10: '0b1

### ISSUE THREAD

**Issue by rafaelreuber** [oracle/python-cx_Oracle#27]
cx_Oracle 6.0b1 breaks SQLAlchemy compatibility. The following code works on previous versions:

```
from sqlalchemy import create_engine
engine = create_engine('oracle+cx_oracle://user:pwd@192.168.100.11:1521/ebs')
```

My Enviroment:

SO: Fedora Linux 25 (64 Bits)
Oracle Instant Client:  11.2.0.4.0 (64 Bits)

Python = 3.5.2 (64 Bits)
cx_Oracle = 6.0b1
SQLAlchemy = 1.1.9
gcc = 6.3.1 20161221 (Red Hat 6.3.1-1)

**Comment by anthony-tuininga** [comment]
Apparently this is due to the fact that SQLAlchemy does not support characters in the version. See this [Stack Overflow question](http://stackoverflow.com/questions/43682595/sqlalchemy-create-engine-issue-with-oracle/43688940). This problem will not exist when cx_Oracle 6 final is released and in the meantime you can adjust SQLAlchemy (to handle the characters) or cx_Oracle (to remove the characters).

**Comment by rafaelreuber** [comment]
Fixed on the SQLAlchemy [1.1.10](https://bitbucket.org/zzzeek/sqlalchemy/issues/3975)
